packages/scalar.aspnetcore/Scalar.AspNetCore.csproj

```diff
@@ -7,8 +7,4 @@
 		<Version>1.1.0</Version>
 	</PropertyGroup>
 
-	<ItemGroup>
-		<PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.5" />
-	</ItemGroup>
-
-</Project>
+</Project>
```

Comment:
Since the package doesn't directly use any of the APIs in `Microsoft.AspNetCore.OpenApi` we can remove this package reference.

Typically, we'd expect that users install the two packages alongside each other:

```
<PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="9.0.0-preview.6" />
<PackageReference Include="Scalar.AspNetCore" Version="1.1.0" />
```

Then they can manage each package version independently.